include/strf/outbuf.hpp

```diff
@@ -476,29 +484,29 @@ class basic_cstr_writer final: public strf::basic_outbuf_noexcept<CharT>
 {
 public:
 
-    basic_cstr_writer(CharT* dest, CharT* dest_end)
+	STRF_HD basic_cstr_writer(CharT* dest, CharT* dest_end)
         : basic_outbuf_noexcept<CharT>(dest, dest_end - 1)
     {
         STRF_ASSERT(dest < dest_end);
     }
 
-    basic_cstr_writer(CharT* dest, std::size_t len)
+	STRF_HD basic_cstr_writer(CharT* dest, std::size_t len)
```

Comment:
Wrong indentation.

Reply:
Here and in some other places in that file.